README.md

```diff
@@ -1,26 +1,14 @@
-# Fx Service Framework
+# :sparkles: Fx [![GoDoc][doc-img]][doc] [![Build Status][ci-img]][ci] [![Coverage Status][cov-img]][cov] [![Report Card][report-card-img]][report-card]
 
-[![GoDoc][doc-img]][doc]
-[![Coverage Status][cov-img]][cov]
-[![Build Status][ci-img]][ci]
-[![Report Card][report-card-img]][report-card]
+An application framework for Go that:
 
-Fx is a flexible, modularized framework for building robust and performant
-services. It takes care of the boilerplate code and lets you focus on your
-application logic.
+* Makes it easy to write composable and testable apps using dependency injection.
+* Removes boilerplate in main and the need for global state and package-level init functions.
+* Eliminates the need for service owners to manually install and manage individual libraries.
```

Comment:
This is an emergent property of modules since they can create an `acmefx.Module` which provides all the libraries necessary for services to "just work" at Acme Inc.